### PR TITLE
[ForYou] Remove ‘works by artists you follow’ rail.

### PR DESCRIPTION
**Pull request by alloy**
Fixes https://github.com/artsy/collector-experience/issues/659
Depends on https://github.com/artsy/metaphysics/pull/814

Skip New Tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Scenes/Home/Components/ForYou/index.tsx [hunk 1]
```diff
@@ -135,6 +135,7 @@ export default createFragmentContainer(
           FOLLOWED_GENES
           GENERIC_GENES
         ]
+        exclude: [FOLLOWED_ARTISTS]
```
Comment (mennenia) [inline src/lib/Scenes/Home/Components/ForYou/index.tsx]: Goodbye 👋

**Comment by mennenia** [comment]
I've added "Skip New Tests" because this PR does not touch upon something that would be tested (the excluding functionality is tested on MP side, as per your PR).

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by alloy** [comment]
Yup, this can be merged then 👍

**Pull request merged**